server/appengine/src/main/java/who/RefreshCaseStatsServlet.java

```diff
@@ -317,11 +319,33 @@ protected void doGet(
       processWhoStats(rows, countryData, globalData);
     }
 
+    CaseStats oldCaseStats = StoredCaseStats.load(JurisdictionType.GLOBAL, "");
```

Comment:
fixPartialLastDayAll should go here. That is, not inside the fetch loop. But before the new delta check.

Reply:
Good call. Fixed.